hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/pages/HollowExplorerPage.java

```diff
@@ -43,12 +43,22 @@ public HollowExplorerPage(HollowExplorerUI ui) {
     public void render(HttpServletRequest req, HttpServletResponse resp, HollowUISession session) throws IOException {
         VelocityContext ctx = new VelocityContext();
 
-        if(ui.getHeaderDisplayString() != null)
-            ctx.put("headerDisplayString", ui.getHeaderDisplayString());
-        
-        if(ui.getCurrentStateVersion() != Long.MIN_VALUE)
+        if (ui.getCurrentStateVersion() != Long.MIN_VALUE)
             ctx.put("stateVersion", ui.getCurrentStateVersion());
-        
+
+        if (ui.getHeaderDisplayMap() != null) {
+            String environment = ui.getHeaderDisplayMap().getOrDefault("environment", "");
```

Comment:
I don't think hollow should know anything about environment or namespace. We could use the keys to mean something special in the UI though, I can walk you through what I mean over a video call

Reply:
As discussed, will change the keys to generic names to avoid confusion in Hollow.